Tests/Amr/Advection_AmrCore/Exec/inputs

```diff
@@ -31,15 +31,17 @@ amr.max_level       = 2       # maximum level number allowed --
 
 amr.ref_ratio       = 2 2 2 2 # refinement ratio between levels
 
+amr.use_bittree = true
+
 # *****************************************************************
 # Control of grid creation
 # *****************************************************************
 # Blocking factor for grid creation in each dimension --
 #   this ensures that every grid is coarsenable by a factor of 8 --
```

Comment:
Does the comment ("every grid is coarsenable by a factor of 8") need updating?

Reply:
Good question. @akashdhruv and @tklos96 Do we still need to set blocking factors and max_grid_size in `inputs_bittree`?